### PR TITLE
CaseClassAddition

### PR DESCRIPTION
**Pull request by solomonmcharo28**
Added Case Class Column

### CODE REVIEW

File: CDC-Data-Reconciliation-Frontend/src/components/Report.jsx [hunk 1]
```diff
@@ -79,6 +79,12 @@ export default function Report({ reportID }) {
       id: "ReasonID",
       footer: (props) => props.column.id,
     },
+    {
+      header: "CaseClassStatus",
+      accessorFn: (row) => row.CaseClassStatus.toString(),
+      id: "CaseClassStatus",
+      footer: (props) => props.column.id,
+    },
```
Comment (Carson21) [inline CDC-Data-Reconciliation-Frontend/src/components/Report.jsx:87]: Maybe we should move this column to be more on the left side of the table?

**Review by Carson21 — CHANGES_REQUESTED** [review]
You forgot to add CaseClassStatus to lines 100, 132, and 144

File: CDC-Data-Reconciliation-Frontend/src/components/Report.jsx (hunk 1 above)
Comment (waffy1901) [inline CDC-Data-Reconciliation-Frontend/src/components/Report.jsx:87]: Maybe move this after EventName?

**Review by Carson21 — APPROVED** [review]
Looks good

**Review by p-zach — COMMENTED** [review]
nvm

**Review by p-zach — APPROVED** [review]

**Pull request merged**